### DISCUSSION TITLE
你知道Coursera嗎？

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#961] [General]
### Coursera是什麼？
Coursera是一個由兩個stanford資工教授開發的線上學習平台，現在已經成為世界上最大的線上學習平台之一，幾乎全世界都在使用它學習，裡面也有很多常春藤的教授開課
### 要錢嗎？
如果你想要在修課結束後得到證書，需要付出一點💰💰，但也可以免費旁聽，只是不能參與作業討論及沒有證書。
但不用擔心，Coursera提供獎學金申請，想知道如何申請就點進去下面這個連結吧
https://medium.com/rosebabble/%E7%B0%A1%E5%96%AE%E5%85%A9%E6%AD%A5%E9%A9%9F-coursera%E8%AA%B2%E7%A8%8B%E7%8D%8E%E5%8A%A9%E5%AD%B8%E9%87%91%E7%94%B3%E8%AB%8B%E6%88%90%E5%8A%9F%E7%B6%93%E9%A9%97%E5%88%86%E4%BA%AB-c6417c9b6129
### 證書有用嗎？
依照每個人的需求不同，證書對於每個人的意義就不一樣，我覺得下面這篇解析的不錯
https://fenixhsu.medium.com/coursera-%E8%AD%89%E6%9B%B8%E5%88%B0%E5%BA%95%E6%9C%89%E6%B2%92%E6%9C%89%E7%94%A8-5f0379c7d1b1
### 個人修課經驗分享
我自己目前有在Coursera上修supply chain的課程，我認為Coursera的修課環境很自由和彈性，相當適合我這種拖延怪，課程或作業的deadline都可以延後，不用擔心在期中期末地獄週被雙面夾擊。
語言的部分不用太擔心，我本來已經做好練英聽的打算，但後來發現其實會有英文字幕，甚至有幾堂課還有中文字幕（但我個人不太推薦依賴中文字幕，因為翻譯有點奇怪，而且不是每部影片都有）。
我目前對於Coursera甚是滿意，提供我一個探索自己興趣的平台，修課的同時也不用顧慮GPA。
### 個人獎學金經驗分享
我一開始很擔心獎學金申請不會過，但其實你只要如實地講出你為什麼想修課以及這門課對於你的幫助，就會通過了。
*貼心小提醒：審核時間通常都是兩星期，所以規劃學習計畫時記得提早申請！

總之很推薦大家上去Coursera看看有什麼感興趣的課程，不只電資，基本上什麼領域的課程都有，而且很多都是名校教授開的，也歡迎大家一起分享修課的經驗

**Comment by dabbie109303501** [comment]
之前有聽過這個平台 但沒有註冊和仔細了解過 看完你的分享讓我想在寒假註冊使用看看！

**Comment by peggy0329** [comment]
一直都知道這個平台，但還沒有實際使用過，主要也是因為怕花了錢上線上課程結果體驗不佳
但他居然有提供獎學金申請，這會大大提升我的使用意願！

**Comment by Tuoresa** [comment]
哇！這是我第一次聽說這個平台
看起來也太具吸引力，感謝有使用經驗的同學分享！
課程的自由選擇性總是能大大提升學習的欲望
選擇自己喜歡的內容加以學習總是令人很興奮哈哈
迫不及待想嘗試看看了

**Comment by Valentinaa0214** [comment]
知道這個平台也有用過，它整合了世界各地不同的課程，讓我們不用實際飛去當地也能線上旁聽，
真的超方便的拉，但我每次都半途而廢，學到一半就放棄，看來這個寒假有事做了，就是把之前的課補完...
謝謝Doreen的詳細推薦(●'◡'●)

**Comment by wenchi9023** [comment]
Coursera超讚!之前有申請SQL課程的獎學金，有個欄位是讓你自己寫希望付的金額是多少，原本我寫課程原價的1/4，但最後的結果是學費全免~~最後還有名校的證書，讚讚

**Comment by AnAnistired** [comment]
我完全沒聽過欸，但這看起來好讚，我下次一定要試試，畢竟某些教授講課常常聽不懂（口語上）
